### PR TITLE
feat: question item options

### PR DESCRIPTION
**Pull request by typeWolffo**
## Overview
- added possibility to assign and edit question options
### Create
image
### Update
image
image

### Notes
- quickfix: added support for external videos on frontend
image

### CODE REVIEW

**Review by piotr-pajak — APPROVED** [review]

File: apps/api/src/lessons/api/lessons.controller.ts
Comment (typeWolffo) [inline apps/api/src/lessons/api/lessons.controller.ts:1]: @wielopolski please take a look

File: apps/api/src/lessons/lessonItems.service.ts [hunk 1]
```diff
@@ -278,6 +279,84 @@ export class LessonItemsService {
     return question;
   }
 
+  async getQuestionOptions(questionId: UUIDType) {
```
Comment (wielopolski) [inline apps/api/src/lessons/lessonItems.service.ts:282]:
```suggestion
  async getQuestionAnswers(questionId: UUIDType) {
```

File: apps/api/src/lessons/lessonItems.service.ts [hunk 2]
```diff
@@ -278,6 +279,84 @@ export class LessonItemsService {
...
+    const options = await this.db
```
Comment (wielopolski) [inline apps/api/src/lessons/lessonItems.service.ts:283]:
```suggestion
    const answers = await this.db
```

File: apps/api/src/lessons/lessonItems.service.ts [hunk 3]
```diff
@@ -278,6 +279,84 @@ export class LessonItemsService {
...
+    options: Array<
+      Partial<{
+        id: string;
+        optionText: string;
+        isCorrect: boolean;
+        position: number;
+      }>
+    >,
+  ) {
+    await this.db.transaction(async (tx) => {
```
Comment (wielopolski) [inline apps/api/src/lessons/lessonItems.service.ts:307]:
```suggestion
    await this.db.transaction(async (trx) => {
```

**Review by wielopolski — APPROVED** [review]
LGTM 🍻 
I left some suggestions

**Pull request merged**